app/src/main/java/org/fossasia/openevent/utils/CircleTransform.java

```diff
@@ -24,7 +24,7 @@ public Bitmap transform(Bitmap source) {
             source.recycle();
         }
 
-        Bitmap bitmap = Bitmap.createBitmap(size, size, source.getConfig());
+        Bitmap bitmap = Bitmap.createBitmap(size, size, Bitmap.Config.ARGB_8888);
```

Comment:
I have to produce it again by using the older code I think, if there is any other way than please suggest me I will show you the stack trace.